client/packages/invoices/src/Prescriptions/DetailView/Payments/PaymentsModal.tsx

```diff
@@ -151,6 +151,8 @@ export const PaymentsModal: FC<PaymentsModalProps> = ({
                   onChange={(_, option) => {
                     if (option) {
                       setInsuranceId(option.value);
+                    } else {
+                      setInsuranceId(null);
                     }
```

Comment:
Personal preference, but for these short condition branches, I like the simpler:
```ts
if (option) setInsuranceId(option.value)
else setInsuranceId(null);
```

Up to you 🤷‍♂️